lib/config/config-initializer.js

```diff
@@ -343,29 +342,26 @@ function promptUser(callback) {
                 return ((answers.source === "guide" && answers.packageJsonExists) || answers.source === "auto");
             }
         }
-    ], earlyAnswers => {
+    ]).then(earlyAnswers => {
 
         // early exit if you are using a style guide
         if (earlyAnswers.source === "guide") {
             if (!earlyAnswers.packageJsonExists) {
                 log.info("A package.json is necessary to install plugins such as style guides. Run `npm init` to create a package.json file and try again.");
-                return;
+                return void 0;
             }
             if (earlyAnswers.styleguide === "airbnb" && !earlyAnswers.airbnbReact) {
                 earlyAnswers.styleguide = "airbnb-base";
             }
-            try {
-                config = getConfigForStyleGuide(earlyAnswers.styleguide);
-                writeFile(config, earlyAnswers.format);
-            } catch (err) {
-                callback(err);
-                return;
-            }
-            return;
+
+            config = getConfigForStyleGuide(earlyAnswers.styleguide);
+            writeFile(config, earlyAnswers.format);
+
+            return void 0;
         }
 
         // continue with the questions otherwise...
-        inquirer.prompt([
+        return inquirer.prompt([
```

Comment:
Any reason we can't return these promises and then outdent all the `then`s in one line? I think it would be easier to read!
```
return inquirer.prompt(a)
    .then(b => {
        // do stuff
        return inquirer.prompt(a);
    })
    .then(b => {
        // do stuff
        return inquirer.prompt(b);
    })
    .then(b => {
        // do stuff
    });